tests/integration/targets/ecs_tag/tasks/main.yml

```diff
@@ -281,6 +281,48 @@
           - taglist.changed == true
           - '"Name" not in taglist.tags'
 
+    # Test tags and tags_propagate with service creation
```

Comment:
Could you please cover the following tests for adding tags and use propagate_tags (where it fits): add tags in check_mode, add tags, check Idempotency by re-tagging, update tags (with purge_tags and without), delete tags, use CamelCase tags as it has been done for example here https://github.com/ansible-collections/community.aws/blob/main/tests/integration/targets/ec2_vpc_nat_gateway/tasks/main.yml#L484.

Reply:
Hey @alinabuzachis ,
Ive covered adding tags while ecs service creation(without check_mode, why is it matter?), covered retagging(should fail), updating tags is not supported using boto3-ecs module as i mentioned in previous comment. deleting a service using this module obviously deletes the tags attached to it.
i'll add the CamelCase tagging.